ghost/members-api/lib/repositories/MemberRepository.js

```diff
@@ -18,7 +18,8 @@ const messages = {
     productNotFound: 'Could not find Product {id}',
     bulkActionRequiresFilter: 'Cannot perform {action} without a filter or all=true',
     tierArchived: 'Cannot use archived Tiers',
-    invalidEmail: 'Invalid Email'
+    invalidEmail: 'Invalid Email',
+    invalidNewsLetterId: 'Cannot subscribe to invalid newsletter {id}'
```

Comment:
```suggestion
    invalidNewsletterId: 'Cannot subscribe to invalid newsletter {id}'
```

(`newsletter` is one word so we don't need to capitalise `Letter` 👍)